webapp/src/shared/components/Input/Input.tsx

```diff
@@ -46,6 +46,7 @@ export default function Input({
           onChange={onChange}
           name={name}
           type={type}
+          autoComplete="off"
```

Comment:
I saw chrome complaining about the password not having auto complete off, but generally, if not for passwords, I don't think it's a problem to have auto complete on. Why do you disagree?

Reply:
Yes, passwords was the main reason... and also, for me, usually is a bit annoying to have the autocomplete display all the crap you've already typed before :sweat_smile: